FIX: CheckIn Update Link Broken in prod

It is not broken in dev

We checked the logs from this morning So we will remove this button and see if it fixes it. 
```
Caused by: org.attoparser.ParseException: Error resolving template [fragments/scrolltopbutton], template might not exist or might not be accessible by any of the configured Template Resolvers (template: “fragments/baseLayout” - line 26, col 6)
[9:29 AM] Trevor Page
Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed: org.thymeleaf.exceptions.TemplateInputException: An error happened during template parsing (template: “class path resource [templates/checkin/read.html]“)] with root cause
```

--- a/src/main/resources/templates/fragments/baseLayout.html
+++ b/src/main/resources/templates/fragments/baseLayout.html
@@ -23,8 +23,8 @@
 <div th:replace="~{fragments/togglescript :: script}"></div>
 <div th:replace="~{fragments/authscript :: script}"></div>
 <div th:replace="~{fragments/themeswitchscript :: script}"></div>
-<div th:replace="~{fragments/scrolltopbutton :: scrollTopButton}"></div>
-<div th:replace="~{fragments/scrolltopbuttonscript :: script}"></div>
+<!--<div th:replace="~{fragments/scrolltopbutton :: scrollTopButton}"></div>-->
+<!--<div th:replace="~{fragments/scrolltopbuttonscript :: script}"></div>-->
 <div th:replace="~{fragments/roleScript :: script}"></div>
 </body>
 </html>

